AudioKit/Common/Internals/Utilities/AVAudioBufferConvenience.swift

```diff
@@ -69,4 +69,62 @@ extension AVAudioPCMBuffer {
         }
         return buffer
     }
+    
+    /// - Returns: The time in seconds of the peak of the buffer or 0 if it failed
+    open var peakTime: Double {
```

Comment:
A function, rather than a property makes more sense here.  Calling a function is a hint to the user that there is work being done.  If it were a catchable value on an immutable object, a property would make sense.

Reply:
yeehaw. keep that whip a crackin

Reply:
hee yaw!